espaloma/graphs/deploy.py

```diff
@@ -88,6 +88,17 @@ def openmm_system_from_graph(
         sys = ff.create_openmm_system(
             g.mol.to_topology(), charge_from_molecules=[g.mol]
         )
+        
+    elif charge_method == "am1-bcc":
+        g.mol.assign_partial_charges()
+        sys = ff.create_openmm_system(
+            g.mol.to_topology(), charge_from_molecules=[g.mol]
```

Comment:
I don't think you want to include `charge_from_molecules` here, do you? You want `ff` to assign AM1-BCC charges, which is specified as part of the openff force field you are using as a base force field (here, openff-1.2.0).

Reply:
after calling `assign_partial_charges()` the charges would be stored in that molecule?

Reply:
Ah, I see! Perhaps we need `am1-bcc`, which calls `assign_partial_charges()`, and a separate `smirnoff` options, which just leaves this up to the `ff.` object to handle charges as specified in the force field?

Reply:
Doesn't `assign_partial_charges` require a `partial_charge_method` argument to be specified?
https://open-forcefield-toolkit.readthedocs.io/en/0.10.1/api/generated/openff.toolkit.topology.Molecule.html#openff.toolkit.topology.Molecule.assign_partial_charges